lib/config.go

```diff
@@ -57,6 +57,24 @@ const (
 	DefaultLogSizeMax = -1
 )
 
+// DefaultCapabilities for the capabilities option in the crio.conf file
+var DefaultCapabilities = []string{
+	"CHOWN",
+	"DAC_OVERRIDE",
+	"FSETID",
+	"FOWNER",
+	"MKNOD",
```

Comment:
I propose we remove MKNOD from default.  This will block the ability for apps to create device nodes.  I don't believe that containers should have this by default.  This is needed for installation not for running containers in production.  The one potential problem with this would be applications creating something like a loopback device.